runtime/slot-composer.js

```diff
@@ -18,6 +18,7 @@ class SlotComposer {
     assert(options.affordance, "Affordance is mandatory");
     assert(options.rootContext, "Root context is mandatory");
 
+    this._containerKind = options.containerKind;
```

Comment:
can the container kind be derived from affordance? or do we need this additional flexibility?
I'm fine either way, just curios.

Reply:
Possibly, but we'd need a more flexible affordance concept. In particular, the `a-frame` requirement is very specific to the environment I'm using for the VR app, it could be done different ways (hence those particles are really `a-frame` not `vr`). We should talk about affordance soon, there are some wrinkles there.

Another options is that the particles themselves could control the container nodes. It could be a property on the particle, or even in the manifest. I'm not sure, what's best long term.

Reply:
Part of me likes the idea of the particle being able to decide, but otoh the system should have some control over the rendering, so probably the way it is now is better.